superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -483,7 +476,7 @@ const ResultSet = ({
     // We need to calculate the height of this.renderRowsReturned()
     // if we want results panel to be proper height because the
     // FilterTable component needs an explicit height to render
-    // react-virtualized Table component
+    // antd Table component
```

Comment:
It's better to just reference the Table component and leave the `antd` part out because in the future we can change our table component to use another implementation and this comment will still be valid.

```suggestion
    // the Table component
```